src/main/java/org/eclipse/microprofile/starter/log/DynamoDBLogger.java

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.log;
+
+import org.eclipse.microprofile.starter.view.EngineData;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import javax.enterprise.context.ApplicationScoped;
+import javax.xml.bind.DatatypeConverter;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+@ApplicationScoped
+public class DynamoDBLogger {
+
+    private static final Logger Log = Logger.getLogger(org.eclipse.microprofile.starter.log.DynamoDBLogger.class.getName());
+    private static final String accessKey = System.getProperty("AWS_ACCESS_KEY_ID");
+    private static final String secretKey = System.getProperty("AWS_SECRET_ACCESS_KEY");
+    private static final String webAppInstanceID = System.getProperty("MP_STARTER_APP_ID", "test-instance");
+    private static final String region = System.getProperty("AWS_REGION", "eu-west-1");
+    private static final String tableName = System.getProperty("AWS_DYNAMODB_TABLE_NAME", "microprofile_starter_log");
+    private static final String logRecordTimestampFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+    private static final String method = "POST";
```

Comment:
Why these constants (method, service, contentType, ...)? It makes it more difficult to actually know/search the value if you look at the code.

Reply:
@rdebusscher It is a relic of rewriting my Python prototype to Java. Gonna amend tomorrow. It won't ever be anything else anyway.